README.md

```diff
@@ -1 +1,17 @@
-# projectportal-email-service
\ No newline at end of file
+# projectportal-email-service
+
+Project Portal email service will run on the production internal cluster, `bkpi-ranch`.
+
+A test version has also been set up on the qa internal cluster, `qbkpi-ranch`.
+
+To access either cluster use the following command 
+```kubectx bkpi-ranch``` or ```kubectx qbkpi-ranch```
+
+To check which namespace you are in, use ```kubens```, make sure you are in the `tpl-projectportal` namespace in either cluster.
+
+The Project Portal email service is a cronjob that is scheduled to run every ... , to check the cronjobs, use ```kubectl get cronjobs```
```

Comment:
I get `No resources found in tpl-projectportal namespace.` back when I run this. Is that expected right now? (If so it's fine!)